core/src/main/java/org/frankframework/errormessageformatters/XslErrorMessageFormatter.java

```diff
@@ -80,7 +80,7 @@ public Message format(String errorMessage, Throwable t, INamedObject location, M
 						log.error("got exception extracting parameters", e);
 					}
 				}
-				result = Message.asMessage(transformerPool.transform(Message.asSource(result), parameterValues));
+				result = Message.asMessage(transformerPool.transform(result.asSource(), parameterValues));
```

Comment:
Wordt de eerste waarde van `result` nu wel goed gesloten?

Reply:
De `originalMessage` bedoel je?

Reply:
`result` wordt teruggegeven dus die ga ik niet sluiten. De `originalMessage` is een input-parameter, waarvan het mij ook niet direct een goed idee lijkt om die als side-effect te sluiten. 
@tnleeuw, wat stel je voor?

Reply:
Ik bedoel de waarde van `result` voordat die wordt overschreven met het resultaat van de `transformerPool.transform()`.

Reply:
Gedaan..kijk maar of je het zo goed vindt! Alles is nu ook verder groen, dus dat mooi.